ProjectEureka/project-eureka-flutter
# PE-76  | Rework Profile Onboarding

**What I did:**

- Removed city and birthday fields.
- Users can now upload a profile picture when they first sign up.
- Users can edit their profile picture and will be able to view it on profile screen and side menu.

**How to test**

1. Pull from my branch
2. Create a new account or use an existing one.
3. Navigate to profile screen and hit edit profile.
4. Take a picture or select from gallery.
5. Make sure picture and your current edits show up on profile screen and side menu.
6. Make sure your picture was uploaded to mongo db and firebase. Match the ids accordingly.
7. Test again, a new account or existing one, which ever you did not test first.
